### ISSUE TITLE
"The system cannot find the path specified."

### ISSUE THREAD

**Issue by osoviejo** [failwyn/WebCompiler#39]
### Installed product versions
- Visual Studio: 2022 Community
- This extension: 1.14.7

### Description
Any attempt to compile a SCSS file generates an error message ("The system cannot find the path specified.").  The error list message identifies the location of the error as the file to be compiled, at line 0.

### Steps to recreate
1. Rolled a fresh .NET Framework 4.8 MVC project, which compiles and runs fine.

2. Added a test.scss file in the Content folder.  It contains a single rule:  a { text-decoration: underline; }.

3. Right-click on test.scss and select "Web Compiler/Re-compile file".  The compilerconfig.json and compilerconfig.json.defaults files are generated by the web compiler.  The compilerconfig.json appears to be correct:
[
  {
    "outputFile": "Content/test.css",
    "inputFile": "Content/test.scss"
  }
]

4. The error noted above is generated on the Error List, and no test.css is created.

### Expected behavior
Generate the css file from the scss file.  This works fine in VS 2019.

Edit: Have quite a few extensions installed.  Tried disabling all others.  No change to the behavior.
Edit: When I created the solution using the VS template, I did not select the option to put the solution and project in same folder.

**Comment by failwyn** [comment]
@osoviejo  Would you mind posting that project, this is working as expected for me.

**Comment by osoviejo** [comment]
Solved.  That you couldn't repro told me it was something environmental on my machine.  This was confirmed when I searched the error message and learned it is specific to cmd.exe.

It's usually a bad path environment variable issue, but in my case it was in the autorun registry entry for cmd.exe.  I have used [clink](https://github.com/chrisant996/clink) for years, but recently updated it.  One of the changes was how its profile directory is specified in the autorun.  I used the recommended "--profile ~\clink", which worked fine for me when launching via a shortcut, but was problematic (apparently) when spawned programmatically.  I just replaced it with an explicit path and all is well.

Thanks for taking a look at this, and my apologies for the wild-goose chase!
